.circleci/config.yml

```diff
@@ -1248,20 +1247,20 @@ workflows:
       - check-kontrol-build:
           requires:
             - contracts-bedrock-build
-      # - contracts-bedrock-tests:
-      #     # Test everything except PreimageOracle.t.sol since it's slow.
-      #     name: contracts-bedrock-tests
-      #     test_list: find test -name "*.t.sol" -not -name "PreimageOracle.t.sol"
-      # - contracts-bedrock-tests:
-      #     # PreimageOracle test is slow, run it separately to unblock CI.
-      #     name: contracts-bedrock-tests-preimage-oracle
-      #     test_list: find test -name "PreimageOracle.t.sol"
-      # - contracts-bedrock-tests:
-      #     # Heavily fuzz any fuzz tests within added or modified test files.
-      #     name: contracts-bedrock-tests-heavy-fuzz-modified
-      #     test_list: git diff origin/develop...HEAD --name-only --diff-filter=AM -- './test/**/*.t.sol' | sed 's|packages/contracts-bedrock/||'
-      #     test_timeout: 1h
-      #     test_profile: ciheavy
+      - contracts-bedrock-tests:
+          # Test everything except PreimageOracle.t.sol since it's slow.
+          name: contracts-bedrock-tests
+          test_list: find test -name "*.t.sol" -not -name "PreimageOracle.t.sol"
+      - contracts-bedrock-tests:
+          # PreimageOracle test is slow, run it separately to unblock CI.
+          name: contracts-bedrock-tests-preimage-oracle
+          test_list: find test -name "PreimageOracle.t.sol"
+      - contracts-bedrock-tests:
+          # Heavily fuzz any fuzz tests within added or modified test files.
+          name: contracts-bedrock-tests-heavy-fuzz-modified
+          test_list: git diff origin/develop...HEAD --name-only --diff-filter=AM -- './test/**/*.t.sol' | sed 's|packages/contracts-bedrock/||'
+          test_timeout: 3h
```

Comment:
So it takes 3 hours to run on our machine?

Reply:
Yes, and it's just a tentative upper limit.